### ISSUE TITLE
Reset tasks generates 504, and then does not reset all the tasks

### ISSUE THREAD

**Issue by garbled1** [meeb/tubesync#365]
If I run reset tasks from the GUI, it starts to reset the tasks, but then hits a 504 issue (which I think is basically a timeout issue) and then only resets the tasks for 1-3 channels.  If I run it on the commandline it works just fine, but takes like 5 minutes or so.

I suspect the timeout in the webserver in the container is not long enough here, and doesn't allow the reset to complete when launched from the gui.  Perhaps either increase the timeout, or fork the reset off, and return immediately, so the reset runs in the background?

**Comment by meeb** [comment]
Yeah this is a big ongoing task discussed in other issues. There are a lot of stacked problems with fixing it and no easy migration path so it's been kicked down the road quite a lot. This is technically a duplicate issue but I can see why you might not have found any of the old open issues covering this.

**Comment by thecreativeone91** [comment]
Is there a CLI command to run this currently? Right now mines pretty broken and unable to be fixed without resetting the tasks.

**Comment by meeb** [comment]
Yep, you found the command!

**Comment by tcely** [comment]
This has already been corrected.

The timeout was raised, and the work being done in the function was reduced.

https://github.com/meeb/tubesync/blob/9289a619ee5c245cc7e4ee2e7f36dd0658fa1ecd/tubesync/sync/views.py#L991-L1011

**Comment by meeb** [comment]
@tcely has done enough work on this to close this for now given any remaining timeouts are likely to be unrelated to any original issues created a while ago. If you still experience timeout issues please create a new issue. Thanks!
